server/controllers/perpMaterialStudentController.js

```diff
@@ -0,0 +1,120 @@
+var formidable = require("formidable");
+const fs = require("fs");
+var dir = __dirname + "/../uploads/qnaStudents/";
+const { StudentPrepMaterial } = require("../models/prepMaterialsStudent.model");
+const { mongo } = require("mongoose");
+
+const path = require("path");
+const responseHelpers = require("../helpers/response.helpers");
+
```

Comment:
- Please fix imports arrange them properly 
- Why are we using `var` please use `let` or `const`

Reply:
sure .. let me make the fixes and I will push by EOD today .

Reply:
Updated the PR.